internal/identity/oidc/google/google.go

```diff
@@ -14,58 +15,63 @@ import (
 	admin "google.golang.org/api/admin/directory/v1"
 
 	"github.com/pomerium/pomerium/internal/httputil"
+	"github.com/pomerium/pomerium/internal/identity"
+	pom_oidc "github.com/pomerium/pomerium/internal/identity/oidc"
 	"github.com/pomerium/pomerium/internal/log"
 	"github.com/pomerium/pomerium/internal/sessions"
 	"github.com/pomerium/pomerium/internal/version"
 )
 
+var _ identity.Authenticator = &GoogleProvider{}
+
+// Name identifies the Google identity provider
+const Name = "google"
+
 const defaultGoogleProviderURL = "https://accounts.google.com"
```

Comment:
based on our discussion above, we can still apply it here https://github.com/pomerium/pomerium/pull/560/files#r398807570